src/business_logic/include/Tplayer.hpp

```diff
@@ -10,19 +11,28 @@ class Tplayer : virtual public Ientity
 {
 private:
 
+    #define STARTING_JUMP_VELOCITY 3
```

Comment:
Best practice is to avoid "define"s.
Consider making it as const member of class:
```suggestion
    const unsigned short m_starting_jump_velocity = 3;
```
or if u need this as GLOBAL_CONSTANT - https://www.youtube.com/watch?v=rQhBECyA6ew

Reply:
done